fastlane_bot/events/async_event_update_utils.py

```diff
@@ -166,16 +220,13 @@ def get_pool_info(
         "tkn0_decimals": tkn0["decimals"],
         "tkn1_symbol": tkn1["symbol"],
         "tkn1_decimals": tkn1["decimals"],
-        "pair_name": tkn0["address"] + "/" + tkn1["address"]
+        "pair_name": tkn0["address"] + "/" + tkn1["address"],
```

Comment:
Minor: since you're using `pool_info["pair_name"].split("/")` below...

...you can also use `"/".join(tkn0["address"], tkn1["address"])` here.

Reply:
Im not a big fan of the join method as I find it less readable